### ISSUE TITLE
Annote a Dataset for NER Trainning

### ISSUE THREAD

**Issue by grayJiaaoLi** [amosproj/amos2024ss08-cloud-native-llm#67]
## User story
1. As a data engineer
2. I want/need to prepare an annotated dataset for NER training
3. So that the NER model can be trained on accurately tagged data

## Acceptance criteria
* Select a suitable amount of Q&A pairs from the HuggingFace
    * Start with 50-100 Q&A Pairs

* Use tools like [Doccano](https://github.com/doccano/doccano) to tag entities according to the defined

* Store the NER training dataset
    * Upload the NER-annotated data in a different directory on HuggingFace
    * Ensure annotated dataset can be used for the NER model

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
